target/linux/ath79/image/generic.mk

```diff
@@ -1732,9 +1732,12 @@ define Device/nec_wg800hp
 endef
 TARGET_DEVICES += nec_wg800hp
 
-define Device/netgear_ex6400_ex7300
+define Device/netgear_ex7300
   $(Device/netgear_generic)
```

Comment:
` $(Device/netgear_generic)` recipe is used for some very old AR724x NETGEAR devices. Could you please try to see if it can work after removing it?

Reply:
Done. Yes, it works, after adding `DEVICE_VENDOR` and `IMAGES`. I have confirmed that upgrading both from sysupgrade and the factory web UI works for both EX7300 and EX7300 v2 series.

Reply:
Happy to hear it works. 

By the way, many developers are working to convert wireless calibration data into NVMEM format. If you are interested, maybe you can try to convert it. Just some information, not something that must be done now. Refer to https://github.com/openwrt/openwrt/commit/3e3e78de1103cb0c5af4cc229152236758c37901

Reply:
No, only EX7300 v2 now. I see that the EX7300 still uses `mtd-cal-data=<&caldata 0x1000>;`.

Reply:
Ah, I see. I've added a commit to convert it now. Checked that the wifi mac addresses are as expected.